AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift

```diff
@@ -616,6 +616,91 @@ public class ConstructorIO: CIOSessionManagerDelegate {
         executeTracking(request, completionHandler: completionHandler)
     }
 
+    /**
+     Track when a user views a quizzes results page
+
+     - Parameters:
+        - quizID: The quiz identifier
+        - quizVersionID: The quiz version identifier
+        - quizSessionID: The quiz session identifier associated with this conversion event
+        - url: The current page url
+        - resultID: The identifier of result set returned by the Constructor quiz response
+        - resultPage: The current page of the results
+        - resultCount: The total numbe of results
+        - sectionName The name of the autocomplete section the results came from
+        - completionHandler: The callback to execute on completion.
+     
+     ### Usage Example: ###
+     ```
+     constructorIO.trackQuizResultsLoaded(quizId: 'coffee-quiz', quizVersionId: '1231244', quizSessionId: '3123', url: 'www.example.com', resultCount: 167)
+     ```
+     */
+    public func trackQuizResultsLoaded(quizID: String, quizVersionID: String, quizSessionID: String, url: String, resultID: String? = nil, resultPage: Int? = nil, resultCount: Int? = nil, sectionName: String? = nil, completionHandler: TrackingCompletionHandler? = nil) {
```

Comment:
I think we want to not expose url here, set a default value in the object. Like we do for browse result load. Since it would likely confuse users.

Reply:
Great catch, I forgot `url` largely isn't applicable to mobile apps